### ISSUE TITLE
The result of each dataset?

### ISSUE THREAD

**Issue by WonderSeven** [narumiruna/efficientnet-pytorch#17]
Could you report the result of each dataset (e.g. CIFAR10, ImageNet)

**Comment by narumiruna** [comment]
I'm still training the model on ImageNet and finding the hyperparameters.

**Comment by WonderSeven** [comment]
The hyperparameters reported in [EfficientNet-Pytorch](https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/examples/imagenet/main.py) do not work well? Since training ImageNet from scratch is time-consuming, I am trying to reproduce the results in CIFAR10 & CIFAR100. But even I have tried many strategies to finetune the EfficientNet, it did not work well on CIFAR10(86.29%) CIFAR100(66.57%).

**Comment by narumiruna** [comment]
The training code is from [PyTorch ImageNet example](https://github.com/pytorch/examples/tree/master/imagenet), the settings of hyperparameters do not work well for my code. Now I am training the model with the gradual warmup.

**Comment by narumiruna** [comment]
I am also trying the settings in the [official implementation](https://github.com/tensorflow/tpu/blob/master/models/official/efficientnet/utils.py#L63)

**Comment by WonderSeven** [comment]
So what are the F1 score and F5 score for your current setting? And what is the GPU hours u need to train?

**Comment by WonderSeven** [comment]
I am trying to reproduce the results reported in [EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks](https://arxiv.org/pdf/1905.11946.pdf) in small scale datasets first. After these, I shall try to train on ImageNet.

**Comment by narumiruna** [comment]
> So what are the F1 score and F5 score for your current setting? And what is the GPU hours u need to train?

The best top1 accuracy I got is ~ 68% (efficientnet_b0) on the validation set.



I just retrained the model, the current top1 acc is 45%
```
Epoch:   3% 8/300 [4:35:28<167:28:14, 2064.71s/it]
Train:  86% 8618/10010 [28:49<04:52,  4.75it/s]
```

**Comment by narumiruna** [comment]
I tried this [config](https://github.com/narumiruna/efficientnet-pytorch/pull/18/files) with 5 epochs gradual warmup and got 94.09% top1 acc.

```
Config(
        use_cuda: True
        data_parallel: True
        num_epochs: 90
        output_dir: experiments/cifar10
        model: {'name': 'efficientnet_b0', 'num_classes': 10, 'in_channels': 3}
        dataset: {'name': 'cifar10_dataloaders', 'root': 'data', 'batch_size': 64, 'image_size': 224, 'num_workers': 16}
        optimizer: {'name': 'SGD', 'lr': 0.025, 'weight_decay': 0.0001, 'momentum': 0.9, 'nesterov': True}
        scheduler: {'name': 'MultiStepLR', 'milestones': [30, 60, 80], 'gamma': 0.1}
        config: configs/cifar10.yaml
        root: None
        resume: None
        backend: nccl
        init_method: tcp://127.0.0.1:23456
        world_size: 1
        rank: 0
)
Files already downloaded and verified
Files already downloaded and verified
Epoch: 100% 90/90 [3:16:01<00:00, 128.30s/it, train loss: 0.3489, train acc: 87.94%, valid loss: 0.1919, valid acc: 94.00%, best valid acc: 94.09]
```

**Comment by eugenelet** [comment]
I'm trying EffNet-B0 on CIFAR100 and couldn't get the reported results. What could I be missing? I'm using the same hyperparameters mentioned.

**Comment by narumiruna** [comment]
> I'm trying EffNet-B0 on CIFAR100 and couldn't get the reported results. What could I be missing? I'm using the same hyperparameters mentioned.

The dataset in reported results is CIFAR10

**Comment by eugenelet** [comment]
I see, seems like EfficientNet is hyperparameter-sensitive.  Do you have any source where I can refer to for CIFAR100?

Edit: Sorry, I just realized that it's an error on my implementation.

**Comment by glenn-jocher** [comment]
@narumiruna hello, any update on reproducing the paper results by training with this repo? Thank you!
